src/Microsoft.AspNetCore.Mvc.Razor/RazorView.cs

```diff
@@ -36,14 +36,14 @@ public class RazorView : IView
         /// </param>
         /// <param name="razorPage">The <see cref="IRazorPage"/> instance to execute.</param>
         /// <param name="htmlEncoder">The HTML encoder.</param>
-        /// <param name="diagnosticSource">The <see cref="DiagnosticSource"/>.</param>
+        /// <param name="diagnosticListener">The <see cref="DiagnosticListener"/>.</param>
         public RazorView(
```

Comment:
>public RazorView [](start = 8, length = 16)

I think this is something we're not likely to have an issue with, I doubt anyone subclassed this.